docs/pages/cheatsheet.adoc

```diff
@@ -323,25 +294,51 @@ No syntax change.
 }
 ----
 
-=== Transfer primitive field
+=== Transfer initializer
+
+The initializer is automatically called when a transfer object is created or selected. It can be used to set the values of the transient fields.
 
 *Syntax:*
 
 [subs="quotes"]
 ----
-[purple]#**field**# &#x5B;[purple]##**required**##] &lt;primitive> &lt;name> &#x5B;([purple]##**<=**##|[purple]##**<=>**##) &lt;expression>] [purple]#**;**#
+[purple]#**initializer**# [purple]#**;**#
+----
+
+=== Transfer destructor
+
+Before the transfer object instance is deleted, the system calls the `destructor`. The destructor is a special action that is used to delete mapped entity or invoke other destruct actions. Transfer objects with no destructor can not be created. The default behavior of the destructor is to delete the mapped entity.
```

Comment:
Please add 'Not supported yet'

Reply:
It must be supported. Empty destructor is the equivalent of D flag in esm.

Reply:
There is a copy paste error: 'Transfer objects with no destructor can not be created' :)

Reply:
There is a copy paste error: 'Transfer objects with no destructor can not be created' :)

Reply:
Ohh... ahh... jesus! :-)

Reply:
destructor removed, delete operation added